ci/release/update-version.sh

```diff
@@ -17,11 +17,13 @@ CURRENT_MAJOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[1]}')
 CURRENT_MINOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[2]}')
 CURRENT_PATCH=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[3]}')
 CURRENT_SHORT_TAG=${CURRENT_MAJOR}.${CURRENT_MINOR}
+CURRENT_UCXPY_VERSION=$(curl -s https://version.gpuci.io/rapids/${CURRENT_SHORT_TAG}`.*)
 
 #Get <major>.<minor> for next version
 NEXT_MAJOR=$(echo $NEXT_FULL_TAG | awk '{split($0, a, "."); print a[1]}')
 NEXT_MINOR=$(echo $NEXT_FULL_TAG | awk '{split($0, a, "."); print a[2]}')
 NEXT_SHORT_TAG=${NEXT_MAJOR}.${NEXT_MINOR}
+NEXT_UCXPY_VERSION=$(curl -s https://version.gpuci.io/rapids/${NEXT_SHORT_TAG}`.*)
```

Comment:
I think this should be `NEXT_UCXPY_VERSION="$(curl -s https://version.gpuci.io/rapids/${NEXT_SHORT_TAG}).*" `

Reply:
Agreed, I now see there were two typos, your suggestion should fix both. Thanks @Ethyling !